Examples/Sources/ControlsExample/ControlsApp.swift

```diff
@@ -22,75 +22,81 @@ struct ControlsApp: App {
+                            }
+                            .padding(.bottom, 20)
+                        #endif
 
-                    #if !canImport(UIKitBackend)
                         VStack {
-                            Text("Checkbox")
-                            Toggle("Toggle me:", active: $exampleCheckboxState)
-                                .toggleStyle(.checkbox)
-                            Text("Currently enabled: \(exampleCheckboxState)")
+                            Text("Toggle switch")
+                            Toggle("Toggle me:", active: $exampleSwitchState)
+                                .toggleStyle(.switch)
+                            Text("Currently enabled: \(exampleSwitchState)")
                         }
-                    #endif
 
-                    VStack {
-                        Text("Slider")
-                        Slider($sliderValue, minimum: 0, maximum: 10)
-                            .frame(maxWidth: 200)
-                        Text("Value: \(String(format: "%.02f", sliderValue))")
-                    }
+                        #if !canImport(UIKitBackend)
+                            VStack {
+                                Text("Checkbox")
+                                Toggle("Toggle me:", active: $exampleCheckboxState)
+                                    .toggleStyle(.checkbox)
+                                Text("Currently enabled: \(exampleCheckboxState)")
+                            }
+                        #endif
+
+                        VStack {
+                            Text("Slider")
+                            Slider($sliderValue, minimum: 0, maximum: 10)
+                                .frame(maxWidth: 200)
+                            Text("Value: \(String(format: "%.02f", sliderValue))")
+                        }
 
-                    VStack {
-                        Text("Text field")
-                        TextField("Text field", text: $text)
-                        Text("Value: \(text)")
-                    }
+                        VStack {
+                            Text("Text field")
+                            TextField("Text field", text: $text)
+                            Text("Value: \(text)")
+                        }
 
-                    Toggle("Enable ProgressView resizability", active: $isProgressViewResizable)
-                    Slider($progressViewSize, minimum: 10, maximum: 100)
-                    ProgressView()
-                        .resizable(isProgressViewResizable)
-                        .frame(width: progressViewSize, height: progressViewSize)
+                        Toggle("Enable ProgressView resizability", active: $isProgressViewResizable)
+                        Slider($progressViewSize, minimum: 10, maximum: 100)
+                        Button("Randomize progress view size") {
+                            progressViewSize = Int.random(in: 10...100)
+                        }
```

Comment:
I think this should be removed. I assume it was probably just used to test for size animation when fixing the AppKitBackend issues?